todo_app/DatabaseHelper.py

```diff
@@ -0,0 +1,67 @@
+from todo_app.Item import Item
+from todo_app.Status import Status
+from pymongo import MongoClient
+from bson import ObjectId
+from datetime import datetime
+
+class DatabaseHelper:
+
+    def __init__(self, url, db):
+        self._db = MongoClient(url)[db]
+
+    def createItem(self, title, description="", due=""):
+        item = {
+            '_id': ObjectId(),
+            'name': title,
+            'desc': description,
+            'due': due,
+            'status_ref': self.getStatusIdForTitle('To Do'),
+            'last_activity': self.getLastActivity()
+        }
+        self._db.items.insert_one(item)
+
+    def getStatusIdForTitle(self, title):
+        return [status.id for status in self.getStatuses() if status.title == title][0]
```

Comment:
Storing status as an ID in the items collection is unnecessary complication. You could have just stored the status directly.

You only need to pull it out into a separate collection if you want to keep the parent items small. Or if there are many properties of the child entity AND it is updated frequently.